tslint.json

```diff
@@ -0,0 +1,3 @@
+{
+  "extends": "gts/tslint.json", "linterOptions": {"exclude": ["src/index.ts"]}
```

Comment:
What's wrong with linting of `src/index.ts`?

Reply:
I think we are fine here now, removed it, thanks

Reply:
+1